enterprise-suite/es-grafana/cluster-exporter.json

```diff
@@ -1,7 +1,44 @@
 [
-   {"graphName": "CPU Percent", "promQL":["node_cpu_percent"]},
-   {"graphName": "Memory Available Percent", "promQL":["node_memory_usable_percent"]},
-   {"graphName": "File System Free Percent", "promQL":["node_filesystem_free_percent"]},
-   {"graphName": "Bytes Received Per Second", "promQL":["sum by (instance) (irate(node_network_receive_bytes[1m]))"]},
-   {"graphName": "Bytes Transmitted Per Second", "promQL":["sum by (instance) (irate(node_network_transmit_bytes[1m]))"]}
+   {
+      "graphName": "CPUs Used",
```

Comment:
As discussed with @dbrinegar offline, I found this name confusing.  I naively thought it meant "number of CPUs dedicated to the container" and _could not_ figure out how that could be derived from `system_seconds`.  I see now that it's a measure of CPU utilization and that it's the standard k8s nomenclature...